### PR TITLE
FIX : [BUG] No padding around 'Select a tab' label in mobile view #10150

### LINKED ISSUE #10150: [BUG] No padding around 'Select a tab' label in mobile view
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

Production

### Version of BioDrop (for example "v1.2.3")

v2.99.4

### Description

The current mobile view of the application displays the '_Select a tab_' label with suboptimal padding, resulting in a less-than-ideal user experience. The inadequate padding around that label on smaller screens can lead to visual clutter and may impact the overall usability of the application.

### Steps to Reproduce

1. Open the application on a mobile device or use a mobile view of the developer tools
2. Navigate to [Community Events](https://www.biodrop.io/events).
3. Observe the spacing around the label (See screenshot below)

### Screenshots

image

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this

Adjust the CSS or styling rules for labels in the mobile view to provide appropriate padding, ensuring a more comfortable and aesthetically pleasing display.

### PR DESCRIPTION
**Pull request by eugene4545**
Closes #10150

## Fixes Issue

  Closes #10150

## Changes proposed (

The select label in the UI was lacking proper padding, which led to alignment issues on smaller screens. To address this, I've added the following Tailwind CSS utility classes to the label's style: "px-6 py-1 mb-2" in the  "select" tag  inside "EventsTab.js" under the "Events" folder

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [] My change requires changes to the documentation.
- [] I have updated the documentation accordingly.
- [] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Before
Screenshot (1213)

After
Screenshot (1215)

### CODE REVIEW

**Review by ThomasCode92 — APPROVED** [review]
Looks good!

**Comment by eugene4545** [comment]
Hey there! Just checking in to see if there's anything else I can do to help get this PR merged. Let me know if there's any concerns about this

File: components/event/EventTabs.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export function EventTabs({ tabs, eventType, onEventTypeChange }) {
           label="Select a tab"
           value={tabs.find((tab) => tab.key === eventType)?.title}
           onChange={(e) => changeTab(e)}
-          className="block w-full rounded-md border-primary-medium-low dark:bg-primary-medium dark:focus:border-secondary-low dark:focus:ring-secondary-low  focus:border-secondary-low focus:ring-secondary-low"
+          className="block w-full rounded-md border-primary-medium-low dark:bg-primary-medium dark:focus:border-secondary-low dark:focus:ring-secondary-low focus:border-secondary-low focus:ring-secondary-low px-3 py-1 mb-2"
```
Comment (ArvindParekh) [inline components/event/EventTabs.js:21]: Looks good! Perhaps consider adding the classes before the `dark:` and `focus:` variants. Doesn't Prettier usually take care of that automatically?

Comment (eugene4545) [reply components/event/EventTabs.js]: Thank you for the feedback, Arvind I've reordered the class names as suggested, with the dark and focus variants now at the end. As for Prettier, it doesn't seem to sort the class names automatically, so I manually rearranged them. Let me know if there's anything else I can do to improve the PR. Thanks again!

Comment (ArvindParekh) [reply components/event/EventTabs.js]: Thanks @eugene4545!
Oh I didn't know about the extra plugin, thanks @ThomasCode92 👍🏻

Comment (eugene4545) [reply components/event/EventTabs.js]: thanks @ThomasCode92 , I have applied the tailwind prettier plugin on the code to rearrange the class name

Comment (eugene4545) [reply components/event/EventTabs.js]: Hey @Arvind I was wondering if there are any remaining issues that need to be addressed.

Comment (ArvindParekh) [reply components/event/EventTabs.js]: No, I think the pr solves the mentioned issue. Let's wait for other to review it and get it merged :)

**Comment by vrushabhgawas14** [comment]
Changes seems good!

**Review by vrushabhgawas14 — APPROVED** [review]
Everything seems good.

**Comment by eugene4545** [comment]
Hey! @kumarsonsoff3   I wanted to follow up on this PR that was opened about a month ago. Are there any outstanding concerns or feedback that I can help address to move this forward? Thank you for your time!

**Comment by eugene4545** [comment]
Hey @kumarsonsoff3 , @ArvindParekh , @SaraJaoude! Hope everyone's doing well! I reached out to @kumarsonsoff3 about my PR a while back and wanted to follow up with the team. I understand you're all busy, and I appreciate your time and efforts. I'm looking for any feedback or updates to help move this along. If anyone has a chance to take a look, I'd really appreciate it! Thanks, everyone!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me. Great collaboration everyone! 👍

--
Sorry for delays, @eugene4545. I was away for a while and might missed your notification. Thanks for being patient, but please avoid tagging multiple maintainers at a time because every maintainer gets notification for each activity on project.

**Comment by ArvindParekh** [comment]
Thank you for your contribution @eugene4545 👍🏻

**Pull request merged**